Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -81,7 +81,11 @@ public struct LinkDestinationSummary: Codable, Equatable {
     public let language: SourceLanguage
     
     /// The relative path to this element.
-    public let path: String
+    ///
+    /// > Note: For elements representing on-page elements, this value will include a fragment component.
```

Comment:
Thanks for documenting this! I wonder if it's a little confusing that the `path` here includes the fragment but since this isn't a change in behavior I'm not concerned.

Reply:
Yes, unfortunately this name doesn't reflect the values that it can have. I would have wanted to rename or even remove it but but of those would be breaking changes. It may still be worth deprecating it though.

Reply:
I decided to deprecate this and rename it to `relativePresentationURL` to better describe what values it represents. It's still encoded as "path" though, to avoid breaking the linkable-entities spec.

Reply:
That makes sense to me. Thank you!